index.ts

```diff
@@ -1,6 +1,8 @@
 import { device, DeviceOptions } from "aws-iot-device-sdk";
 import { Subject, Observer, Observable } from "rxjs";
-import * as AWS from "aws-sdk";
+import * as Iot from "aws-sdk/clients/iot";
+import { config as AWSConfig } from "aws-sdk/global";
```

Comment:
@mrarnolds Won't importing from /global still include everything or did I misunderstand you?

Reply:
Global and root have some minor differences, global imports only /core, but root takes it all. I guess core is anyway needed.

Reply:
Alright, in that case it looks good. Will merge and publish